### PR TITLE
bundle version tag into docker container binary

### PR DESCRIPTION
**Pull request by joschahenningsen**

### CODE REVIEW

**Comment by joschahenningsen** [comment]
This sets the version tag for the docker container as well @alexanderstephan 

https://test.live.mm.rbg.tum.de/healthcheck

**Comment by joschahenningsen** [comment]
Honestly no idea what that commit ref on the testserver is, probably some github action `on:pr` rebase/merge-commit stuff. But as long as it's unique :shrug:

File: .github/workflows/ts-deploy.yml [hunk 1]
```diff
@@ -50,5 +52,6 @@ jobs:
           context: .
           pull: true
           push: true
+          build-args: version=${{ github.sha }}
```
Comment (alexanderstephan) [inline .github/workflows/ts-deploy.yml]: We could calculate the shorter commit sha with 
```
run: echo "SHORT_SHA=`echo ${GITHUB_SHA} | cut -c1-8`" >> $GITHUB_ENV
```

**Review by alexanderstephan — CHANGES_REQUESTED** [review]
Awesome, thanks for the quick PR! :heart: Maybe we could shorten the SHA to make the distinction easier.

**Review by alexanderstephan — APPROVED** [review]
Looks good, thanks! :rocket:

**Pull request merged**